packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/overview/edit-name/edit-name.controller.js

```diff
@@ -0,0 +1,53 @@
+export default class OctaviaLoadBalancerEditNameCtrl {
+  /* @ngInject */
+  constructor(OctaviaLoadBalancerService, Alerter, $translate) {
+    this.isLoading = false;
+    this.OctaviaLoadBalancerService = OctaviaLoadBalancerService;
+    this.Alerter = Alerter;
+    this.$translate = $translate;
+  }
+
+  $onInit() {
+    this.name = this.loadbalancer.name;
+  }
+
+  cancel() {
+    this.trackCancel();
+    this.goBack();
```

Comment:
```suggestion
    return this.goBack();
```

Reply:
I don't see the purpose of returning here: cancel is used in the template where return value is ignored